openmc/weight_windows.py

```diff
@@ -216,13 +220,13 @@ def energy_bounds(self, bounds):
         self._energy_bounds = np.asarray(bounds)
 
     @property
-    def num_energy_bins(self):
+    def num_energy_bins(self) -> int:
         if self.energy_bounds is None:
             raise ValueError('Energy bounds are not set')
         return self.energy_bounds.size - 1
 
     @property
-    def lower_ww_bounds(self):
+    def lower_ww_bounds(self) -> Iterable[Real]:
```

Comment:
When this attribute is set, it is converted to a numpy array, so this should probably be:
```suggestion
    def lower_ww_bounds(self) -> np.ndarray:
```